templates/create_meeting.html

```diff
@@ -1,16 +1,15 @@
 {% extends 'base.html' %}
+{% include "modals.html" %}
 {% block content %}
-<!DOCTYPE html>
-<html>
-<head>
-<meta name="viewport" content="width=device-width, initial-scale=1">
+<!-- <meta name="viewport" content="width=device-width, initial-scale=1"> -->
+<!-- This is a change-->
 <style>
 body {font-family: Arial, Helvetica, sans-serif;}
 * {box-sizing: border-box;}
 
 input[type=text], select, textarea {
   width: 100%;
-  padding: 10px;
+  padding: 5px;
   border: 1px solid #ccc;
   border-radius: 4px;
   box-sizing: border-box;
```

Comment:
The <style> tag should really be in the head for it to be supported by all browsers. I recommend that page-specific styles be moved to base.html--and, if a style needs to be confined to a single page or element, use element IDs or classes to give the CSS scope.

Since the style code block already exists, I won't take this into account when reviewing this pull request.